sdk/core/core-amqp/src/errors.ts

```diff
@@ -598,14 +676,16 @@ export function translate(err: AmqpError | Error): MessagingError {
     // translate
     const condition = (err as any).code;
     const description = (err as Error).message;
```

Comment:
The casting to `Error` is not needed here anymore as the `if (isAmqpError)` check a few lines above ensures that by the time we get here, the error is guaranteed to be of class `Error`

Reply:
Cleaned up castings that aren't needed.